src/components/Card/Card.css

```diff
@@ -2,3 +2,32 @@
   height: 30vh;
   width: auto;
 }
+
+.card-container {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
```

Comment:
The card container has no direct styling and only serves to orient the contents
of each card. As explained in Feed.css, applying flex-row to the container
prevents any context issues caused by the native centering in React.